modules/quality_control/php/quality_control.class.inc

```diff
@@ -272,17 +278,14 @@ class Quality_Control extends \NDB_Menu_Filter
                 "SELECT ID FROM mri_scan_type WHERE Scan_type=:scan_type",
                 $query_params
             );
+            //base on the raisinbread instrument - mri_parameter_form table
+            // there are only "3d_t1w_Scan_Done" and "2d_flair_Scan_Done"
             if (substr("$scan_type", 0, 2) == "t1") {
-                $scans_done[$scan_type] = "T1_Scan_done";
-            } elseif (substr("$scan_type", 0, 3) == "t2w") {
-                $scans_done[$scan_type] = "T2_Scan_done";
-            } elseif (substr("$scan_type", 0, 3) == "dti") {
-                $scans_done[$scan_type] = "dti_Scan_done";
+                $scans_done[$scan_type] = "3d_t1w_Scan_Done";
             } else {
-                $scans_done[$scan_type] = $scan_type . "_Scan_done";
```

Comment:
The X_Scan_Done in the mri_parameter_form instrument is supposed to match the mri_scan_type table. You should just update the mri_parameter_form to match raisinbread